docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -166,81 +192,44 @@ For reference, the following is the list of codes in this category:
 
-#### 1.2.5 fhir-server-config.json properties
-The following properties are available to configure searchParameterFilter:
+##  2 Re-index
+Reindexing is implemented as a custom operation that tells the server to read a set of resources and replace the existing search parameters with those newly extracted from the resource body.
+
+The `$reindex` operation can be invoked via an HTTP(s) POST to `[base]/$reindex`. By default, the operation will select 10 resources and re-extract their search parameters values based on the current configuration of the server. The operation supports the following parameters to control the behavior:
+|name|type|description|
+|----|----|-----------|
+|`_tstamp`|string|Reindex any resource not previously reindexed before this timestamp. Format as a date YYYY-MM-DD or time YYYY-MM-DDTHH:MM:DDZ.|
+|`_resourceCount`|integer|The maximum number of resources to reindex in this call. If this number is too large, the processing time might exceed the transaction timeout and fail.|
 
-##### 1.2.5.1 Default property values
-| Property Name                 | Default value   |
-|-------------------------------| ----------------|
-|`fhirServer/searchParameterFilter`|`"*": ["*"]`|
+To aid the re-indexing process, the IBM FHIR Server team has introduced some new features into the fhir-bucket resource-loading tool for driving the reindex. The fhir-bucket tool uses a thread-pool to make concurrent POST requests to the IBM FHIR Server `$reindex` custom operation.
 
-##### 1.2.5.2 Property attributes
-| Property Name                 | Tenant-specific? | Dynamic? |
-|-------------------------------|------------------|----------|
-|`fhirServer/searchParameterFilter`|Y|Y|
+To run the reindex step, see this example (using PostgreSQL):
 
-In general, the configuration properties for a particular tenant are stored in the `<WLP_HOME>/wlp/usr/servers/fhir-server/config/<tenant-id>/fhir-server-config.json` file, where `<tenant-id>` refers to the tenant's "short name" or tenant id. The global configuration is considered to be associated with a tenant named `default`, so those properties are stored in the `<WLP_HOME>/wlp/usr/servers/fhir-server/config/default/fhir-server-config.json` file. Similarly, tenant-specific search parameters are found at `<WLP_HOME>/wlp/usr/servers/fhir-server/config/<tenant-id>/extension-search-parameters.json` whereas the global/default extension search parameters are at `<WLP_HOME>/wlp/usr/servers/fhir-server/config/default/extension-search-parameters.json`. Search parameters are handled like a single configuration property; providing a tenant-specific file will override the global/default extension search parameters.
+```
+JAR="/path/to/fhir-bucket-4.5.0-cli.jar"
+
+java \
+  -Djava.util.logging.config.file=logging.properties \
+  -jar "${JAR}" \
+  --db-type postgresql \
+  --fhir-properties fhir.properties \
+  --tenant-name "YOUR-TENANT-NAME" \
```

Comment:
is tenant-name actually used with postgres?

Reply:
I don't think it is, but I copied these argument right from what @punktilious put in our 4.5.0 release notes.  I know we have #1640 for cleaning up the arguments needed by this utility, so hopefully we can clean up this doc as part of that.